<launch>
    <arg name="pgm_map_name" default="map" />
    <arg name="display_point_cloud" default="true" />
    <node name="map_server" pkg="map_server" type="map_server" args="$(find ros_navigation_humanoid)/maps/$(arg pgm_map_name).yaml"/>

    <group if="$(arg display_point_cloud)">
        <node name="point_cloud_display_node" pkg="ros_navigation_humanoid" type="point_cloud_display_node">
            <param name="publish_topic" value="map_point_cloud" />
            <param name="frame_id"  value="map" />
            <param name="file_path"  value="$(find ros_navigation_humanoid)/maps/$(arg pgm_map_name).pcd" />
            <param name="z_min" value="0.0" />
            <param name="z_max" value="2.0" />
            <param name="voxel_filte_size" value="0.05" />
            <param name="pub_rate" value="0.5" />
        </node>
    </group>

</launch>